src/utilities/UtilitiesSql.i

```diff
@@ -41,4 +41,32 @@
 
 // order that these are loaded matters, e.g TimeSeries in data needs to know about DateTime in time
 %include <utilities/sql/SqlFile.i>
+
+%{
```

Comment:
http://www.swig.org/Doc4.0/SWIG.html#SWIG_nn44 seems to place it in a `%init %{ ... }%` block. Is this on purpose that this doesn't have the `%init `  specifier?

Reply:
It was on purpose. The documentation and the code samples do not agree with each other. It says some targets do not support `%init` so you might need to use a `static` variable, then goes on to put the `static` inside of an `%init`! If the target doesn't support `%init` then that should have no impact, right?

I figured I would do the ultimately most generic option because of C# on Windows, just to be safe.